src/AppBundle/Tests/Controller/PostControllerApiTest.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+namespace AppBundle\Tests\Controller;
+
+use Lakion\ApiTestCase\JsonApiTestCase;
+use Symfony\Component\HttpFoundation\Response;
+
+class PostControllerApiTest extends JsonApiTestCase
```

Comment:
This only a test class. It will be moved to DemoBunle and user should delete whole of it. We agreed about leaving it empty.